src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -2121,7 +2121,7 @@ internal enum ErrorCode
         WRN_DuplicateAnalyzerReference = 9067,
         ERR_FileTypeNonUniquePath = 9068,
         ERR_FilePathCannotBeConvertedToUtf8 = 9069,
-        //ERR_ReadOnlyNotSuppAsParamModDidYouMeanIn = 9070,
+        ERR_RequiredMembersAttributeErrors = 9070,
```

Comment:
I think we shouldn't reuse codes from a released version of the compiler. Consider using one of the error codes marked as `// available` below